nautobot_plugin_nornir/plugins/credentials/nautobot_secrets.py

```diff
@@ -40,6 +40,8 @@ def _get_access_type_value(device_obj):
     """
     if PLUGIN_CFG.get("use_config_context", {}).get("secrets"):
         access_type_str = device_obj.get_config_context()["nautobot_plugin_nornir"]["secret_access_type"].upper()
+        if access_type_str == "HTTP(S)":
```

Comment:
```suggestion
        if access_type_str is ["HTTP(S)", "HTTP"]:
```
Perhaps we try to be user friendly here? Just to try and catch a misunderstanding from a user

Reply:
Sure, makes sense. Should it be `is` or `in` though?

Reply:
Updated 🧠 ☁️